packages/js/src/Flare.ts

```diff
@@ -22,6 +22,7 @@ export class Flare {
         stage: '',
         maxGlowsPerReport: 30,
         reportingUrl: 'https://reporting.flareapp.io/api/reports',
+        reportBrowserExtensionErrors: false,
```

Comment:
What was the exact reason again we didn't set this to true by default? Scared about hiding exceptions which we shouldn't?

Reply:
This sets _reporting_ to false by default, so we'll be hiding them.